api/spec/model/retro_spec.rb

```diff
@@ -82,6 +82,20 @@
     end
   end
 
+  describe '#magic_link_enabled' do
+    it 'behaves like a boolean' do
+      retro = Retro.new
+
```

Comment:
Should it check here if magic link is disabled by default?

Reply:
I was a bit weary of testing default behavior here.  A non-virtual attribute would actually be "nil" by default because it's unset.  But there's no way to really make this nil since it's a reflection of whether the value is set or not.

Reply:
thats fair, plus its tested in E2E I believe